src/main/java/dev/slimevr/vr/trackers/TrackerPosition.java

```diff
@@ -103,12 +102,26 @@ public static Optional<TrackerPosition> getByDesignation(String designation) {
 			return Optional.empty();
 		}
 
-		// Support old configs. leg was renamed to knee.
-		if (designation.equalsIgnoreCase("body:left_leg"))
-			designation = "body:left_knee";
-		if (designation.equalsIgnoreCase("body:right_leg"))
-			designation = "body:right_knee";
-
+		// Support old configs.
+		if (
+			designation.equalsIgnoreCase("body:left_leg")
+				|| designation.equalsIgnoreCase("body:left_knee")
+		) {
+			designation = "body:left_upper_leg";
+		} else if (
+			designation.equalsIgnoreCase("body:right_leg")
+				|| designation.equalsIgnoreCase("body:right_knee")
+		) {
+			designation = "body:right_upper_leg";
+		} else if (designation.equalsIgnoreCase("body:left_ankle")) {
+			designation = "body:left_lower_leg";
+		} else if (designation.equalsIgnoreCase("body:right_ankle")) {
+			designation = "body:right_lower_leg";
+		} else if (designation.equalsIgnoreCase("body:left_forearm")) {
+			designation = "body:left_lower_arm";
+		} else if (designation.equalsIgnoreCase("body:right_forearm")) {
+			designation = "body:right_lower_arm";
+		}
```

Comment:
Just realized I forgot a check here.
Need to check for "body:left_upperarm" and replace it by "body:left_upper:arm" (same with right!!!)

Reply:
Not many people used the upperarm, since it's only in early, beta, builds.
What would be even better would be handling when `designation` doesn't correspond to any TrackerPosition. Currently it just throws an exception and kinda breaks IIRC.

Reply:
Forgot we can push everywhere except main so I fixed it :3 
Altho handling when `designation` doesn't correspond to anything would be a good idea... I just checked it and it throws an exception and prevents any more trackers from appearing in GUI.

Reply:
If you fixed it, then approve the merge :nya_a: